_resources/port1.0/group/python-1.0.tcl

```diff
@@ -368,6 +376,13 @@ proc python_add_dependencies {} {
         } else {
             depends_lib-delete port:python${python.version}
             depends_lib-append port:python${python.version}
+            if {${python.version} >= 313} {
+                # Python 3.13 uses atomics, which is not supported in old Xcode compilers.
+                # Python.framework/Versions/3.13/include/python3.13/cpython/pyatomic.h:543:4:
+                # error: #error "no available pyatomic implementation for this platform/compiler"
+                # error: command '/usr/bin/gcc-4.2' failed with exit code 1
+                compiler.c_standard 2011
```

Comment:
This will downgrade the requested standard if a newer one was set in the Portfile.

Reply:
Good point. Should probably have a followup to make it conditional (if possible).

Reply:
Hmm that's not right then. It should be added to the top level like `default compiler.c_standard {2011}`?